### DISCUSSION TITLE
Transform Seurat Object to ArchR

### DISCUSSION THREAD

**Discussion by CeliaJuarillo** [GreenleafLab/ArchR#2111] [Questions / Documentation]
Hi, 
I'm new to ArchR and I have been trying to convert my Seurat Object into ArchR. I don't even know if this is possible, but I have a multiome object from Seurat and Signac with RNA and ATAC information. This object has integrated information from multiple samples.

Is there a way to use this information directly into ArchR? There is a pipeline to follow in case I can create a ArchR object and then add the metadata from the clusters from my Seurat object?

**Comment by alifarhat40** [comment]
Hi @CeliaJuarillo Did you ever figure this out? I am looking to do the same.
